front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -209,24 +209,22 @@ const IdeasWithFiltersSidebar = ({
 
   return (
     <Container id="e2e-ideas-container">
-      <Box display="flex" justifyContent="space-between" mb="8px">
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        mb={showViewButtons ? '8px' : '16px'}
+      >
         {inputTerm && (
-          <Title variant="h4" as="h2" mt="auto" mb="auto" color="tenantText">
-            {formatMessage(messages.ideasFilterSidebarTitle, {
-              numberIdeas: ideasCount,
-              inputTerm: formatMessage(
-                getInputTermMessage(inputTerm, {
-                  idea: messages.ideas,
-                  option: messages.options,
-                  project: messages.projects,
-                  question: messages.questions,
-                  issue: messages.issues,
-                  contribution: messages.contributions,
-                  proposal: messages.proposals,
-                  initiative: messages.initiatives,
-                  petition: messages.petitions,
-                })
-              ),
+          <Title
+            fontSize="m"
+            variant="h4"
+            as="h2"
+            my="auto"
+            color="tenantText"
+            style={{ fontWeight: 600 }}
```

Comment:
Let's adjust `Title` if a certain font-weight is missing, rather than using `style` directly (or leave out the font-weight adjustment if that looks fine and you want to reduce work).

Reply:
Good idea! Added it to the Text/Title FontWeight type :) I'll post about it in the FE chapter channel.

Also [created this PR](https://github.com/CitizenLabDotCo/citizenlab/pull/9721) to adjust pas occurrences where the style was used to set the 600 font weight.